File: python/movie.py
class Movie:
    CHILDREN = 2
    NEW_RELEASE = 1
    REGULAR = 0

    def __init__(self, title, price_code):
        self.title = title
        self.price_code = price_code

    def get_price_code(self):
        return self.price_code

    def set_price_code(self, arg):
        self.price_code = arg

    def get_title(self):
        return self.title

File: cpp/Movie.cpp
#include "Movie.h"


Movie::Movie(const std::string& title, int price_code) {
    this->title_ = title;
    this->price_code_ = price_code;
}

int Movie::get_price_code() const
{
    return this->price_code_;
}

std::string Movie::get_title() const
{
    return this->title_;
}

void Movie::set_price_code(int price_code) {
    this->price_code_ = price_code;
}